modules/tinymce/src/themes/silver/main/ts/ui/core/color/ColorSwatch.ts

```diff
@@ -138,11 +138,10 @@ const getToolTipText = (editor: Editor, format: ColorFormat, lastColor: string)
     return format === 'forecolor' ? 'Text color' : 'Background color';
   }
 
-  const tooltipPrefix = format === 'forecolor' ? 'Text color {0}' : 'Background color {0}';
   const colors = getColors(Options.getColors(editor, format), format, false);
   const colorText = Arr.find(colors, (c) => c.value === lastColor).getOr({ text: '' }).text;
 
-  return editor.translate([ tooltipPrefix, editor.translate(colorText) ]);
+  return editor.translate([ `${format === 'forecolor' ? 'Text color' : 'Background color'} {0}`, colorText ]);
 };
```

Comment:
The reason for doing this `return editor.translate([ tooltipPrefix, editor.translate(colorText) ]);` was colorText would be translated first before being substituted into the placeholder, changing this would lead to the `Text color` translated, but the color left untranslated.

Reply:
And that's probably true of the `Tooltip.getTooltipText()` function as well. It shouldn't hurt anything to have the double-translate call, it's a fairly simple object lookup. I suggest pulling this commit out of the PR.

Reply:
That's a good catch, reverted the change.